File: sample01/src/main/java/net/jworkflow/sample01/steps/Goodbye.java
package net.jworkflow.sample01.steps;

import net.jworkflow.kernel.interfaces.StepBody;
import net.jworkflow.kernel.models.*;

public class Goodbye implements StepBody {

    @Override
    public ExecutionResult run(StepExecutionContext context) {
        System.out.println("Goodbye world");
        return ExecutionResult.next();
    }
    
}

File: sample01-kotlin/src/main/java/net/jworkflow/sample01/steps/Goodbye.kt
package net.jworkflow.sample01.steps
import net.jworkflow.kernel.interfaces.StepBody
import net.jworkflow.kernel.models.*

class Goodbye:StepBody {
    @Override
    fun run(context:StepExecutionContext?):ExecutionResult? {
        System.out.println("Goodbye world")
        return ExecutionResult.next()
    }
}